client/src/elements/molecules/item-list-entry.js

```diff
@@ -26,12 +30,34 @@ export class ItemListEntry {
     });
   }
 
+  toggleOptionDropdown() {
+    this.optionsDropdown.classList.toggle(
+      "item-list-entry__options-dropdown--show"
+    );
+  }
+
+  activateItem() {
+    this.item.activate();
+  }
+
+  deactivateItem() {
+    this.item.deactivate();
+  }
+
   deleteItem() {
     this.item.delete().then(() => {
-      this.element.addEventListener("transitionend", () => {
```

Comment:
why remove the transition? it was the only animation we had in Q-editor :-) makes me sad.

Reply:
Added back in https://github.com/nzzdev/Q-editor/pull/133/commits/f653a255ba161bb98e01d0f914ddbf204da4bd41